app.js

```diff
@@ -26,6 +26,14 @@ if (!process.env.RACKSPACE_USERNAME
   throw new Error('Required parameters not provided from the environment');
 }
 
+routes.client.createContainer({
```

Comment:
That's fair, but you should still wrap the server.listen inside of this work to ensure we're not a functional server until this has completed.